configuration.nix

```diff
@@ -46,6 +46,7 @@ isMaximal: {
 
       nix.enable = true;
 
+      assembly.enable = isMaximal;
```

Comment:
I don't think this is needed. Assembly is not common, and the asm LSP is a major hit-or-miss. Lets set this to false.

Reply:
sounds good. Will set to false